tests/test_managed_data.py

```diff
@@ -60,6 +65,38 @@ def config(pytestconfig):
     # teardown tasks, if any
 
 
+def test_create_workspace(config):
+    data = _api.create_workspace(
+        name="test-" + str(uuid.uuid4())[:6],
+        client=config["client"],
+    )
+
+    CreateWorkspaceResponse(**data)
```

Comment:
we check the structure of the response by coercing it into that model. in future these APIs will return typed info, so we won't have to do this